python/ray/data/tests/test_dataset.py

```diff
@@ -1613,163 +1613,6 @@ def test_tensors_in_tables_to_torch_variable_shaped(
-        }
-    )
-    arr2 = np.arange(num_items, 2 * num_items).reshape(shape).astype(np.float)
-    df2 = pd.DataFrame(
-        {
-            "one": TensorArray(arr2),
-            "two": TensorArray(arr2 + 1),
-            "label": [4, 5, 6],
-        }
-    )
-    df = pd.concat([df1, df2])
-    ds = ray.data.from_pandas([df1, df2])
-    ds = maybe_pipeline(ds, pipelined)
-    tfd = ds.to_tf(
-        label_column="label",
-        output_signature=(
-            tf.TensorSpec(shape=(None, 2, 2, 2, 2), dtype=tf.float32),
-            tf.TensorSpec(shape=(None,), dtype=tf.float32),
-        ),
-        batch_size=2,
-    )
-    features, labels = [], []
-    for batch in tfd.as_numpy_iterator():
-        features.append(batch[0])
-        labels.append(batch[1])
-    features, labels = np.concatenate(features), np.concatenate(labels)
-    values = np.stack([df["one"].to_numpy(), df["two"].to_numpy()], axis=1)
-    np.testing.assert_array_equal(values, features)
-    np.testing.assert_array_equal(df["label"].to_numpy(), labels)
-
-
-@pytest.mark.parametrize("pipelined", [False, True])
-def test_tensors_in_tables_to_tf_mix(ray_start_regular_shared, pipelined):
-    import tensorflow as tf
-
-    outer_dim = 3
-    inner_shape = (2, 2, 2)
-    shape = (outer_dim,) + inner_shape
-    num_items = np.prod(np.array(shape))
-    arr = np.arange(num_items).reshape(shape).astype(np.float)
-    df1 = pd.DataFrame(
-        {
-            "one": TensorArray(arr),
-            "two": [1, 2, 3],
-            "label": [1.0, 2.0, 3.0],
-        }
-    )
-    arr2 = np.arange(num_items, 2 * num_items).reshape(shape).astype(np.float)
-    df2 = pd.DataFrame(
-        {
-            "one": TensorArray(arr2),
-            "two": [4, 5, 6],
-            "label": [4.0, 5.0, 6.0],
-        }
-    )
-    df = pd.concat([df1, df2])
-    ds = ray.data.from_pandas([df1, df2])
-    ds = maybe_pipeline(ds, pipelined)
-    tfd = ds.to_tf(
-        label_column="label",
-        feature_columns=[["one"], ["two"]],
-        output_signature=(
-            (
-                tf.TensorSpec(shape=(None, 2, 2, 2), dtype=tf.float32),
-                tf.TensorSpec(shape=(None, 1), dtype=tf.float32),
-            ),
-            tf.TensorSpec(shape=(None,), dtype=tf.float32),
-        ),
-        batch_size=2,
-    )
-    col1, col2, labels = [], [], []
-    for batch in tfd.as_numpy_iterator():
-        col1.append(batch[0][0])
-        col2.append(batch[0][1])
-        labels.append(batch[1])
-    col1 = np.concatenate(col1)
-    col2 = np.squeeze(np.concatenate(col2), axis=1)
-    labels = np.concatenate(labels)
-    np.testing.assert_array_equal(col1, np.sort(df["one"].to_numpy()))
-    np.testing.assert_array_equal(col2, np.sort(df["two"].to_numpy()))
-    np.testing.assert_array_equal(labels, np.sort(df["label"].to_numpy()))
-
-
-@pytest.mark.parametrize("pipelined", [False, True])
-def test_tensors_in_tables_to_tf_variable_shaped(ray_start_regular_shared, pipelined):
```

Comment:
Now that the variable-shaped tensor column (simple ragged tensors) PR is unreverted, could we add test coverage of variable-shaped tensors back? https://github.com/ray-project/ray/pull/29071

Reply:
Discussed offline. We'll do this in follow-up PRs:
1. #29135
2. #29136